protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -45,8 +45,8 @@ def split_input_data(dirac, filelist, n_file_per_job):
 Script.registerSwitch(
     "", "upload_analysis_cfg=", "If True (default), upload analysis configuration file"
 )
-Script.registerSwitch("", "dry=", "If True do not submit job (default: False)")
-Script.registerSwitch("", "test=", "If True submit only one job (default: False)")
+Script.registerSwitch("", "dry", "If True do not submit job (default: False)")
```

Comment:
If this fixes it (meaning that it becomes like argparse's `store_true` option) I think we should change the description to just 

> do not submit job / submit only one job

Reply:
good point